utils_cv/detection/dataset.py

```diff
@@ -155,104 +235,32 @@ def split_train_test(
 
         return train, test
 
-    def show_ims(
-        self, rows: int = 1, cols: int = 3, rand: bool = False
-    ) -> None:
+    def show_ims(self, rows: int = 1, cols: int = 3) -> None:
```

Comment:
Removed 'rand' since was not used anyway and not obvious how to e.g. plot first N images.

Reply:
Fyi, show_ims() throws an error when setting cols = 1